sqlmesh/core/model/kind.py

```diff
@@ -288,6 +309,7 @@ def metadata_hash_values(self) -> t.List[t.Optional[str]]:
             *super().metadata_hash_values,
             str(self.batch_size) if self.batch_size is not None else None,
             str(self.forward_only),
+            str(self.on_schema_change),
```

Comment:
this is a breaking change, you're gonna need to upgrade sqlglot or have a migration

Reply:
> you're gonna need to upgrade sqlglot

FYI sqlglot was bumped to v23.14.0 recently and this hasn't been released yet, see https://github.com/TobikoData/sqlmesh/compare/v0.94.0...main.